tests/fdb/test_fdb_mac_expire.py

```diff
@@ -0,0 +1,258 @@
+import logging
+import pytest
+import time
+
+from common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+class TestFdbMacExpire:
+    """
+        TestFdbMacExpire Verifies FDb aging timer is respected
+
+        The test updates fdb_aging_time value, restarts swssconfig in order to pickup the new value,
+        populated the FDB table with dummy MAC entry, and then waits for fdb_aging_time and makes sure
+        FDB entry with dummy MAC is cleared.
+    """
+    DUMMY_MAC_PREFIX = "00:11:22:33:44"
+    FDB_INFO_FILE = "/root/fdb_info.txt"
+    POLLING_INTERVAL_SEC = 15
+    FDB_AGING_TIMER_EXPIRE_WINDOW = 15
+
+    def __getFdbTableCount(self, duthost, mac):
+        """
+            Gets number of FDB table entries containing mac entry.
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+                mac (str): MAC value to search for in the FDB table
+
+            Returns:
+                (int) representing the number of the FDB entries containing th eMAC address
+        """
+        result = duthost.shell("show mac | grep {0} | wc -l".format(mac))
+        pytest_assert(result["rc"] == 0, "Failed to obtain mac address with error '{0}'".format(result["stderr"]))
```

Comment:
If run the above shell module without 'module_ignore_errors=True' argument, then exception will be raised if rc!=0. The pytest_assert would not get a chance to be executed here when rc!=0.